src/canvasMachine.tsx

```diff
@@ -186,32 +186,39 @@ export const canvasMachine = canvasModel.createMachine({
       }),
     },
     'elkGraph.UPDATE': {
-      actions: canvasModel.assign({
-        elkGraph: (_, e) => e.elkGraph,
-      }),
+      actions: [
+        canvasModel.assign({
+          elkGraph: (_, e) => e.elkGraph,
+        }),
+        send('FIT_TO_VIEW'),
```

Comment:
Does this trigger when the source updates through `inspect` mode?

Reply:
Yes - whenever the elk graph calculation updates, this will update.